extensions/agroal/spi/src/main/java/io/quarkus/agroal/deployment/JdbcDriverBuildItem.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.agroal.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class JdbcDriverBuildItem extends MultiBuildItem {
```

Comment:
Let's have some minimal Javadoc for this one - it seems like a pretty Build Item :)

Reply:
Done.